cypress/support/calendar.js

```diff
@@ -0,0 +1,112 @@
+import localforage from 'localforage';
+import { Link, MultiColumnListCell, Pane, Modal, Button } from '../../interactors';
+
+Cypress.Commands.add('openCalendarSettings', (isLoggedIn = true) => {
+  if (!isLoggedIn) {
+    cy.loginAsAdmin();
+  }
+  cy.visit('settings/calendar');
+});
+
+Cypress.Commands.add('createCalendar', (testCalendarRequestBody, callback = null) => {
```

Comment:
don't use cypress.comands.add for it please. move it inside fragments and use just common function

e2e/support/fragments/calendar folder